.github/workflows/tests.yaml

```diff
@@ -1,25 +1,20 @@
-on: [push, pull_request]
+on: [push, pull request]
 
 jobs:
   tests:
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout repository and submodules
-      uses: actions/checkout@v2
-      with:
-        submodules: recursive
+      - uses: actions/checkout@v3
```

Comment:
It seems it's not aligned correctly, this could explain why it's not running in the PR, also it's missing the `name` in the steps for checkout.

Reply:
Addressed in #37